examples/exec.py

```diff
@@ -0,0 +1,89 @@
+import time
+
+from kubernetes import config
+from kubernetes.client import configuration
+from kubernetes.client.apis import core_v1_api
+from kubernetes.client.rest import ApiException
+
+config.load_kube_config(
+    context="gke_cloud-kubernetes-dev_us-central1-f_mehdy-cluster")
+configuration.assert_hostname = False
+api = core_v1_api.CoreV1Api()
+name = 'busybox-test2'
+
+resp = None
+try:
+    resp = api.read_namespaced_pod(name=name,
+                                   namespace='default')
+except ApiException as e:
+    if e.status != 404:
+        print("Unknown error: %s" % e)
+        exit(1)
+
+if not resp:
+    print("Pod %s does not exits. Creating it..." % name)
+    pod_manifest = {
+        'apiVersion': 'v1',
+        'kind': 'Pod',
+        'metadata': {
+            'name': name
+        },
+        'spec': {
+            'containers': [{
+                'image': 'busybox',
+                'name': 'sleep',
+                "args": [
+                    "/bin/sh",
+                    "-c",
+                    "while true;do date;sleep 5; done"
+                ]
+            }]
+        }
+    }
+    resp = api.create_namespaced_pod(body=pod_manifest,
+                                     namespace='default')
+    while True:
+        resp = api.read_namespaced_pod(name=name,
+                                       namespace='default')
+        if resp.status.phase != 'Pending':
+            break
+        time.sleep(1)
+    print "Done."
+
+
+# calling exec and wait for response.
+exec_command = [
+    '/bin/sh',
+    '-c',
+    'echo This message goes to stderr >&2; echo This message goes to stdout']
+resp = api.connect_get_namespaced_pod_exec(name, 'default',
+                                           command=exec_command,
+                                           stderr=True, stdin=False,
+                                           stdout=True, tty=False)
+print "Response: " + resp
```

Comment:
There are many cases that we can cover. We can add more examples later (and feel free to add more).